### PR TITLE
[CON-1636] Create DocuSign Monitor source doc

### PR DESCRIPTION
**Pull request by keyur-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Comment by JainM6** [comment]
@keyur-sumo : It looks like you have made any changes to the existing workflow?

**Comment by keyur-sumo** [comment]
@JainM6 : No, I haven't changed the workflow.

**Comment by keyur-sumo** [comment]
@kimsauce @nmohantyctr @stacykor : Can you please help with review?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+3. Locate and copy **User ID** available under **My Account Information**. <br/> <img src={useBaseUrl('img/send-data/docusign-user-id.png')} alt="<docusign-user-id.png>" width="600" />
+:::note
+Remember, you will need User ID while creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
+
+### App
+
+You must first create an app to get integration key and configure RSA Key Pair and Redirect URI. This key is required to get access token which will be used to authenticate DocuSign API. To create an app follow the steps below:
+1. Login to **DocuSign** account
+2. Go to **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:47]:
```suggestion
2. Go to the **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+7. Click on **GENERATE RSA** under **Service Integration** to create new private and public key pair. <br/> <img src={useBaseUrl('img/send-data/docusign-generate-rsa.png')} alt="<docusign-generate-rsa.png>" width="600" />
+8. Copy **Private Key** from dialog and close the dialog. <br/> <img src={useBaseUrl('img/send-data/docusign-private-key.png')} alt="<docusign-private-key.png>" width="600" />
+:::note
+Remember, you will need RSA private key while creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
+9. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="600" />
+:::note
+Remember, you will need redirect uri while requesting [application consent](#app-consent).
+:::
+10. Click on **SAVE** to finish new app configuration.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:63]:
```suggestion
10. Click **SAVE** to finish new app configuration.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+5. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category metadata is stored in a searchable field called `_sourceCategory`.
+6. (Optional) **Fields**. Click the **+Add Field** link to define the fields you want to associate. Each field needs a name (key) and value.
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+7. In **DocuSign Environment**, choose the environment of your DocuSign account.
+8. In **User ID**, enter the User ID of your account. See [User ID](#user-id) section to know your User ID.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:106]:
```suggestion
8. In **User ID**, enter the User ID of your account. See [User ID](#user-id) section to help find your User ID.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+
+## Prerequisites
+
+To collect customer event data from the DocuSign Monitor, you must meet the following criteria:
+* Administrator access to an organization: To call the DocuSign Monitor endpoint, you must impersonate a user with administrator access to your organization.
+* Your organization must have access to DocuSign Monitor to call the endpoint.
+* Integration Key: An integration key identifies your integration and links to its configuration values. [Create an integration key](https://developers.docusign.com/platform/configure-app/#how-to-get-an-integration-key)
+* Redirect Uri: The redirect URI is the URI (URL) to which DocuSign will redirect the browser after authentication. [Set a redirect URI](https://developers.docusign.com/platform/configure-app/#how-to-set-a-redirect-uri)
+* RSA Key Pair: [Add the RSA key pair](https://developers.docusign.com/platform/configure-app/#add-the-rsa-key-pair)
+* Application Consent: Refer Step-1 of [Get an access token with JWT Grant](https://developers.docusign.com/platform/auth/jwt/jwt-get-token/)
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:22]:
```suggestion
* **Administrator access to an organization**. To call the DocuSign Monitor endpoint, you must impersonate a user with administrator access to your organization.
   * Your organization must have access to DocuSign Monitor to call the endpoint.
* **Integration Key**. An integration key identifies your integration and links to its configuration values. [Create an integration key](https://developers.docusign.com/platform/configure-app/#how-to-get-an-integration-key)
* **Redirect Uri**: The redirect URI is the URI (URL) to which DocuSign will redirect the browser after authentication. [Set a redirect URI](https://developers.docusign.com/platform/configure-app/#how-to-set-a-redirect-uri)
* **RSA Key Pair**. [Add the RSA key pair](https://developers.docusign.com/platform/configure-app/#add-the-rsa-key-pair)
* **Application Consent**. Refer Step-1 of [Get an access token with JWT Grant](https://developers.docusign.com/platform/auth/jwt/jwt-get-token/)
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+
+## Data sources
+
+The DocuSign API integration retrieves events data every 5 minutes.
+
+## Configuration
+
+In this configuration, you will set up the DocuSign account and configure it to be authorized and authenticated to use customer event data from DocuSign Monitor API.
+To obtain the DocuSign auth token and customer event data, you will need the following parameters:
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:31]:
```suggestion
In this configuration, you will set up the DocuSign account and configure it to be authorized and authenticated to use customer event data from DocuSign Monitor API. To obtain the DocuSign auth token and customer event data, you will need the following parameters:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+### User ID
+
+To get the User ID, follow the steps below:
+1. Login to **DocuSign** account
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:36]:
```suggestion
1. Log in to **DocuSign** account.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+2. Go to **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:37]:
```suggestion
2. Go to the **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+:::
+
+### App
+
+You must first create an app to get integration key and configure RSA Key Pair and Redirect URI. This key is required to get access token which will be used to authenticate DocuSign API. To create an app follow the steps below:
+1. Login to **DocuSign** account
+2. Go to **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
+3. Navigate to **ADD APP AND INTEGRATION KEY**. <br/> <img src={useBaseUrl('img/send-data/docusign-add-app-integration-key.png')} alt="<docusign-add-app-integration-key.png>" width="600" />
+4. Enter value for **App Name** in a dialog box, and click on **CREATE APP**. <br/> <img src={useBaseUrl('img/send-data/docusign-create-app.png')} alt="<docusign-create-app.png>" width="600" />
+5. After app creation, you will be redirected to app configuration page. Copy **Integration Key**. <br/> <img src={useBaseUrl('img/send-data/docusign-integration-key.png')} alt="<docusign-integration-key.png>" width="600" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:50]:
```suggestion
5. After creating your app, you'll be redirected to the app configuration page. Copy **Integration Key**. <br/> <img src={useBaseUrl('img/send-data/docusign-integration-key.png')} alt="<docusign-integration-key.png>" width="600" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+:::note
+Remember, you will need integration key while requesting [application consent](#app-consent) and creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:52]:
```suggestion
You'll need to provide your integration key while requesting [application consent](#app-consent) and creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+:::
+7. Click on **GENERATE RSA** under **Service Integration** to create new private and public key pair. <br/> <img src={useBaseUrl('img/send-data/docusign-generate-rsa.png')} alt="<docusign-generate-rsa.png>" width="600" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:54]: Please fix numbered steps here (goes from step 5 to 7).

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+9. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="600" />
+:::note
+Remember, you will need redirect uri while requesting [application consent](#app-consent).
+:::
+10. Click on **SAVE** to finish new app configuration.
+
+### App Consent
+
+For requesting application consent, follow below steps:
+1. Copy below url based on your account environment in browser and press enter, replace **INTEGRATION_KEY**, **REDIRECT_URI** with the values you copied at step 5 and step 8 of [App](#app) section respectively.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:68]: The steps in the App section are out of order. Can you confirm "step 5 and step 8" are the correct ones you want to refer to?

Comment (keyur-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:67]: Have verified and step #s are correct.

**Comment by keyur-sumo** [comment]
@kimsauce: updated the doc as per review comments.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 12]
```diff
@@ -0,0 +1,189 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+DocuSign pioneered the development of e-signature technology, and today DocuSign helps organizations connect and automate how they prepare, sign, act on, and manage agreements. As part of the DocuSign Agreement Cloud, DocuSign offers eSignature: the world's #1 way to sign electronically on practically any device, from almost anywhere, at any time.
+
+The DocuSign provides a secure endpoint to receive customer event data from the [DocuSign Monitor API](https://developers.docusign.com/docs/monitor-api/reference/monitor/dataset/getstream/). DocuSign Monitor helps organizations protect their agreements with round-the-clock activity tracking. The Monitor API delivers this activity tracking information directly to existing security stacks or data visualization tools—enabling teams to detect unauthorized activity, investigate incidents, and quickly respond to verified threats.
+
+## Prerequisites
+
+To collect customer event data from the DocuSign Monitor, you must meet the following criteria:
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:16]:
```suggestion
To collect event data from the DocuSign Monitor, you must meet the following criteria:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 13]
```diff
@@ -0,0 +1,189 @@
...
+## Configuration
+
+In this configuration, you will set up the DocuSign account and configure it to be authorized and authenticated to use customer event data from DocuSign Monitor API. To obtain the DocuSign auth token and customer event data, you will need the following parameters:
+
+### User ID
+
+To get the User ID, follow the steps below:
+1. Log in to **DocuSign** account.
+2. Go to the **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
+3. Locate and copy **User ID** available under **My Account Information**. <br/> <img src={useBaseUrl('img/send-data/docusign-user-id.png')} alt="<docusign-user-id.png>" width="600" />
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:37]:
```suggestion
3. Locate and copy the **User ID** available under **My Account Information**. <br/> <img src={useBaseUrl('img/send-data/docusign-user-id.png')} alt="<docusign-user-id.png>" width="600" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 14]
```diff
@@ -0,0 +1,189 @@
...
+
+To collect customer event data from the DocuSign Monitor, you must meet the following criteria:
+* **Administrator access to an organization**. To call the DocuSign Monitor endpoint, you must impersonate a user with administrator access to your organization.
+   * Your organization must have access to DocuSign Monitor to call the endpoint.
+* **Integration Key**. An integration key identifies your integration and links to its configuration values. [Create an integration key](https://developers.docusign.com/platform/configure-app/#how-to-get-an-integration-key)
+* **Redirect Uri**. The redirect URI is the URI (URL) to which DocuSign will redirect the browser after authentication. [Set a redirect URI](https://developers.docusign.com/platform/configure-app/#how-to-set-a-redirect-uri)
+* **RSA Key Pair**. [Add the RSA key pair](https://developers.docusign.com/platform/configure-app/#add-the-rsa-key-pair)
+* **Application Consent**. Refer Step-1 of [Get an access token with JWT Grant](https://developers.docusign.com/platform/auth/jwt/jwt-get-token/)
+
+## Data sources
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:24]:
```suggestion
## Data Source
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 15]
```diff
@@ -0,0 +1,189 @@
...
+
+### App
+
+You must first create an app to get integration key and configure RSA Key Pair and Redirect URI. This key is required to get access token which will be used to authenticate DocuSign API. To create an app follow the steps below:
+1. Log in to your **DocuSign** account.
+2. Go to the **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
+3. Navigate to **ADD APP AND INTEGRATION KEY**. <br/> <img src={useBaseUrl('img/send-data/docusign-add-app-integration-key.png')} alt="<docusign-add-app-integration-key.png>" width="600" />
+4. Enter value for **App Name** in a dialog box, and click on **CREATE APP**. <br/> <img src={useBaseUrl('img/send-data/docusign-create-app.png')} alt="<docusign-create-app.png>" width="600" />
+5. After creating your app, you'll be redirected to the app configuration page. Copy **Integration Key**. <br/> <img src={useBaseUrl('img/send-data/docusign-integration-key.png')} alt="<docusign-integration-key.png>" width="600" />
+:::note
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:50]:
```suggestion

:::note
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 16]
```diff
@@ -0,0 +1,189 @@
...
+You'll need to provide your integration key while requesting [application consent](#app-consent) and creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
+6. Click on **GENERATE RSA** under **Service Integration** to create new private and public key pair. <br/> <img src={useBaseUrl('img/send-data/docusign-generate-rsa.png')} alt="<docusign-generate-rsa.png>" width="600" />
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:53]:
```suggestion

6. Click on **GENERATE RSA** under **Service Integration** to create new private and public key pair. <br/> <img src={useBaseUrl('img/send-data/docusign-generate-rsa.png')} alt="<docusign-generate-rsa.png>" width="600" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 17]
```diff
@@ -0,0 +1,189 @@
...
+1. Log in to your **DocuSign** account.
+2. Go to the **My Apps & Keys** page. <br/> <img src={useBaseUrl('img/send-data/docusign-home.png')} alt="<docusign-home.png>" width="600" />
+3. Navigate to **ADD APP AND INTEGRATION KEY**. <br/> <img src={useBaseUrl('img/send-data/docusign-add-app-integration-key.png')} alt="<docusign-add-app-integration-key.png>" width="600" />
+4. Enter value for **App Name** in a dialog box, and click on **CREATE APP**. <br/> <img src={useBaseUrl('img/send-data/docusign-create-app.png')} alt="<docusign-create-app.png>" width="600" />
+5. After creating your app, you'll be redirected to the app configuration page. Copy **Integration Key**. <br/> <img src={useBaseUrl('img/send-data/docusign-integration-key.png')} alt="<docusign-integration-key.png>" width="600" />
+:::note
+You'll need to provide your integration key while requesting [application consent](#app-consent) and creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:52]:
```suggestion
:::

```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 18]
```diff
@@ -0,0 +1,189 @@
...
+6. Click on **GENERATE RSA** under **Service Integration** to create new private and public key pair. <br/> <img src={useBaseUrl('img/send-data/docusign-generate-rsa.png')} alt="<docusign-generate-rsa.png>" width="600" />
+7. Copy **Private Key** from dialog and close the dialog. <br/> <img src={useBaseUrl('img/send-data/docusign-private-key.png')} alt="<docusign-private-key.png>" width="600" />
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:54]:
```suggestion

7. Copy **Private Key** from dialog and close the dialog. <br/> <img src={useBaseUrl('img/send-data/docusign-private-key.png')} alt="<docusign-private-key.png>" width="600" />
```

Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:54]:
```suggestion
7. Copy **Private Key** from dialog and close the dialog. <br/> <img src={useBaseUrl('img/send-data/docusign-private-key.png')} alt="<docusign-private-key.png>" width="600" />

```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 19]
```diff
@@ -0,0 +1,189 @@
...
+:::note
+You'll need to provide RSA private key while creating the [DocuSign Cloud-to-Cloud Source](#set-up-docusign-source).
+:::
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:57]:
```suggestion
:::

```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 20]
```diff
@@ -0,0 +1,189 @@
...
+8. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="600" />
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:58]:
```suggestion
8. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="600" />

```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 21]
```diff
@@ -0,0 +1,189 @@
...
+:::note
+You'll need redirect uri while requesting [application consent](#app-consent).
+:::
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:61]:
```suggestion
:::

```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 22]
```diff
@@ -0,0 +1,189 @@
...
+* **Integration Key**. An integration key identifies your integration and links to its configuration values. [Create an integration key](https://developers.docusign.com/platform/configure-app/#how-to-get-an-integration-key)
+* **Redirect Uri**. The redirect URI is the URI (URL) to which DocuSign will redirect the browser after authentication. [Set a redirect URI](https://developers.docusign.com/platform/configure-app/#how-to-set-a-redirect-uri)
+* **RSA Key Pair**. [Add the RSA key pair](https://developers.docusign.com/platform/configure-app/#add-the-rsa-key-pair)
+* **Application Consent**. Refer Step-1 of [Get an access token with JWT Grant](https://developers.docusign.com/platform/auth/jwt/jwt-get-token/)
+
+## Data sources
+
+The DocuSign API integration retrieves events data every 5 minutes.
+
+## Configuration
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:28]:
```suggestion
## Setup and Configuration
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 23]
```diff
@@ -0,0 +1,189 @@
...
+:::
+8. Click on **ADD URI** under **Redirect URIs** and enter redirect uri to add new redirect uri. <br/> <img src={useBaseUrl('img/send-data/docusign-redirect-uri.png')} alt="<docusign-redirect-uri.png>" width="600" />
+:::note
+You'll need redirect uri while requesting [application consent](#app-consent).
+:::
+9. Click **SAVE** to finish new app configuration.
+
+### App Consent
+
+For requesting application consent, follow below steps:
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:66]:
```suggestion
For requesting application consent, follow the steps below:
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 24]
```diff
@@ -0,0 +1,189 @@
...
+1. Copy below url based on your account environment in browser and press enter, replace **INTEGRATION_KEY**, **REDIRECT_URI** with the values you copied at step 5 and step 8 of [App](#app) section respectively.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:67]:
```suggestion
1. Copy the below URL based on your account environment in the browser and press enter, replace **INTEGRATION_KEY**, and **REDIRECT_URI** with the values you copied at step 5 and step 8 of [App](#app) section respectively.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 25]
```diff
@@ -0,0 +1,189 @@
...
+   - For Development (Demo) Environment, https://account-d.docusign.com/oauth/auth?response_type=code&scope=signature&client_id=INTEGRATION_KEY&redirect_uri=REDIRECT_URI
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:68]:
```suggestion
   * For Development (Demo) Environment, https://account-d.docusign.com/oauth/auth?response_type=code&scope=signature&client_id=INTEGRATION_KEY&redirect_uri=REDIRECT_URI
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 26]
```diff
@@ -0,0 +1,189 @@
...
+   - For Producstion Environment, https://account.docusign.com/oauth/auth?response_type=code&scope=signature&client_id=INTEGRATION_KEY&redirect_uri=REDIRECT_URI
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:69]:
```suggestion
   * For Production Environment, `https://account.docusign.com/oauth/auth?response_type=code&scope=signature&client_id=INTEGRATION_KEY&redirect_uri=REDIRECT_URI`
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 27]
```diff
@@ -0,0 +1,189 @@
...
+2.  Click on **ALLOW ACCESS** to provide your consent. This should redirect you to **REDIRECT_URI**.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:70]:
```suggestion
2.  Click **ALLOW ACCESS** to provide your consent. This should redirect you to **REDIRECT_URI**.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 28]
```diff
@@ -0,0 +1,189 @@
...
+| `name` | String | Yes | Type the desired name of the Source and it must be unique per Collector. This value is assigned to the `metadata field _source`.  | modifiable |
+| `description` | String  | No | Type the description of the Source. | modifiable |
+| `category` | String | No | Type the category of the source. This value is assigned to the metadata field `_sourceCategory`. | modifiable |
+| `fields` | JSON Object | No | JSON map of key-value fields (metadata) to apply to the Collector or Source. Use the boolean field `_siemForward` to enable forwarding to SIEM. | modifiable |
+| `env` | String | Yes | Environment of the DocuSign application. | modifiable |
+| `userId` | String | Yes | User ID of the DocuSign account. | modifiable |
+| `integrationKey` | String | Yes | Integration Key of the app. | modifiable |
+| `rsaPrivateKey` | String | Yes | RSA Private Key for the app. | modifiable |
+
+### JSON example
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:168]:
```suggestion
### JSON Example
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 29]
```diff
@@ -0,0 +1,189 @@
...
+* Method: POST
+* Example endpoint: `https://api.sumologic.com/api/v1/collectors/{collector_id}/sources/{source_id}/action/restart`.
+
+<details><summary>Which API endpoint should I use?</summary>
+
+{@import ../../../reuse/api-endpoints.md}
+
+</details>
+
+### JSON configuration
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:145]:
```suggestion
### JSON Configuration
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Some formatting changes.

**Comment by keyur-sumo** [comment]
@JainM6 : updated documents and images as per comments. Can you please review again?

**Comment by keyur-sumo** [comment]
@JainM6 @kimsauce : As all review comments are resolved, can you please help publish the document?

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 30]
```diff
@@ -0,0 +1,194 @@
+---
+id: docusign-source
+title: DocuSign
+sidebar_label: DocuSign
+description: This document explains how to collect customer event data from the DocuSign and send it to Sumo Logic.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+DocuSign pioneered the development of e-signature technology, and today DocuSign helps organizations connect and automate how they prepare, sign, act on, and manage agreements. As part of the DocuSign Agreement Cloud, DocuSign offers eSignature: the world's #1 way to sign electronically on practically any device, from almost anywhere, at any time.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:10]: Tone down the marketing language (our focus should be what the Sumo Logic app can do for customers)

```suggestion
DocuSign pioneered the development of e-signature technology, and today DocuSign helps organizations connect and automate how they prepare, sign, act on, and manage agreements. As part of the DocuSign Agreement Cloud, DocuSign offers eSignature, allowing you to sign documents electronically from most devices.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 31]
```diff
@@ -0,0 +1,194 @@
+---
+id: docusign-source
+title: DocuSign
+sidebar_label: DocuSign
+description: This document explains how to collect customer event data from the DocuSign and send it to Sumo Logic.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:5]:
```suggestion
description: Learn how to collect customer event data from the DocuSign and send it to Sumo Logic.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 32]
```diff
@@ -0,0 +1,190 @@
...
+On the [Collection page](/docs/manage/health-events#collection-page), the Health and Status for Sources is displayed. Use [Health Events](/docs/manage/health-events.md) to investigate issues with collection. You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.
+
+## Set up DocuSign Source
+
+When you create a DocuSign Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure the DocuSign source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+2. On the Collectors page, click **Add Source** next to a Hosted Collector.
+3. Select **DocuSign** icon.  <br/>  <img src={useBaseUrl('img/send-data/docusign-icon.svg')} alt="docusign-icon.svg" width="120" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:99]:
```suggestion
3. Select **DocuSign** icon.  <br/><img src={useBaseUrl('img/send-data/docusign-icon.svg')} alt="docusign-icon.svg" width="60" />
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md [hunk 33]
```diff
@@ -0,0 +1,190 @@
...
+
+## Set up DocuSign Source
+
+When you create a DocuSign Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Configure a Hosted Collector](/docs/send-data/hosted-collectors/configure-hosted-collector).
+
+To configure the DocuSign source:
+1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
+2. On the Collectors page, click **Add Source** next to a Hosted Collector.
+3. Select **DocuSign** icon.  <br/><img src={useBaseUrl('img/send-data/docusign-icon.svg')} alt="docusign-icon.svg" width="60" />
+4. Enter a **Name** to display for the Source in the Sumo Logic web application. The description is optional. <br/>   <img src={useBaseUrl('img/send-data/docusign-config-main.png')} alt="docusign-config-main.png" width="600" />
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/docusign-source.md:100]:
```suggestion
4. Enter a **Name** to display for the Source in the Sumo Logic web application. The description is optional. <br/>   <img src={useBaseUrl('img/send-data/docusign-config-main.png')} alt="docusign-config-main.png" width="400" />
```

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
> As all review comments are resolved, can you please help publish the document?

@keyur-sumo sure thing! I just made a few formatting tweaks. Pushing it out now.

**Pull request merged**